packages/core/src/lib/actions/invalid-link-destroyed.action.ts

```diff
@@ -1,9 +1,9 @@
-import { BaseAction } from './base.action';
+import { BaseMouseAction } from './base-mouse.action';
 import { PortModel } from '../models/port.model';
 import { NodeModel } from '../models/node.model';
 import { LinkModel } from '../models';
 
-export class InvalidLinkDestroyed extends BaseAction {
+export class InvalidLinkDestroyed extends BaseMouseAction {
```

Comment:
why link destruction is related to BaseMouseAction?
I know it was related to the BaseAction as everything extended it, but should it stay this way?